content/docs/engine-dev/tools/o3de-cli/project-export/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+linkTitle: Project Export
+title: Project Export
+description: High Level Developer Overview for the Project Export command.
+---
+
+***The Project Export*** command enables shipping of O3DE projects into distributable games. It is designed to fulfill arbitrary needs in the export process, and comes with standard export scripts for the majority of users.
+
+{{< note >}}
+You can find a presentation of the Project Export feature on the official O3DE Youtube channel: https://www.youtube.com/watch?v=fV1gmxiRHdY
```

Comment:
-> YouTube

Reply:
do you mean change "Youtube channel:" to "Youtube:"?